### PR TITLE
Disable analytics in OSS code

### PR DESCRIPTION
**Pull request by TrangPham**
See for context: https://github.com/greatexpectationslabs/infrastructure/pull/311/files#r1854502407

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -46,4 +46,7 @@ RUN apt-get remove -y \
     cpp-12 \
     cpp
 
+# Disable analytics in OSS
+RUN export GX_ANALYTICS_ENABLED=false
```
Comment (anthonyburdi) [inline Dockerfile:50]: Any reason to not use the `ENV` instruction? https://docs.docker.com/reference/dockerfile/#env

**Review by anthonyburdi — APPROVED** [review]
Just a question, but if this works no need to block.

File: Dockerfile (hunk 1 above)
Comment (billdirks) [inline Dockerfile:50]: nit: Can you capitalize `False` to match our docs?

**Review by billdirks — APPROVED** [review]
LGTM, thanks!

File: Dockerfile (hunk 1 above)
Comment (billdirks) [reply Dockerfile:50]: Oh good catch, you should use ENV. I think RUN just sets it for this session when building the image but ENV will set it in the resultant image. At least according to this: https://stackoverflow.com/questions/33379393/docker-env-vs-run-export

**Pull request merged**